drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/main/java/org/drools/workbench/screens/scenariosimulation/webapp/client/dropdown/KogitoRuntimeDmnAssetsDropdownProviderImpl.java

```diff
@@ -23,6 +23,7 @@
 import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
 
+import org.drools.workbench.screens.scenariosimulation.kogito.client.dropdown.ScenarioKogitoCreationAssetsDropdownProvider;
```

Comment:
I think it is an unused dependency

Reply:
@danielezonca It is the implemented interface.